server/embed/config.go

```diff
@@ -904,6 +905,10 @@ func (cfg *configYAML) configFromFile(path string) error {
 	if cfg.SelfSignedCertValidity == 0 {
 		cfg.SelfSignedCertValidity = 1
 	}
+
+	if cfg.WatchProgressNotifyIntervalJSON.Duration != 0 {
+		cfg.WatchProgressNotifyInterval = cfg.WatchProgressNotifyIntervalJSON.Duration
+	}
```

Comment:
Suggestion to drop the `.Duration`

```suggestion
	if cfg.WatchProgressNotifyIntervalJSON != 0 {
		cfg.WatchProgressNotifyInterval = cfg.WatchProgressNotifyIntervalJSON
	}
```